src/test/java/com/wikia/webdriver/common/remote/operations/DiscussionsCategoryOperations.java

```diff
@@ -36,6 +43,17 @@ public void deleteCategory(CategoryContext context) {
     new DeleteCategory(user).execute(context);
   }
 
+  public ArrayList<CategoryPill.Data> getCategoriesFromSite(final String siteId, User user) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1319)